src/traces/azureFunctionsHook.ts

```diff
@@ -13,19 +13,17 @@ export class AzureFunctionsHook {
     constructor() {
         try {
             this._functionsCoreModule = require("@azure/functions-core");
+            // Only v3 of Azure Functions library is supported right now. See matrix of versions here:
+            // https://github.com/Azure/azure-functions-nodejs-library
+            const funcProgModel = this._functionsCoreModule.getProgrammingModel();
```

Comment:
Can this return null?

Reply:
It shouldn't but if that happens the catch will take care of it. I don't believe we want to do anything if we cannot get the programming model.